tests/api/activitypub/webfinger.test.js

```diff
@@ -170,12 +187,22 @@ describe('activitypub:webfinger', () => {
       const resource = `acct:${name}@${publicHost}`
       const res = await publicReq('get', `${endpoint}${resource}`)
       const { subject, aliases, links } = res
-      res.should.be.an.Object()
       subject.should.equal(resource)
       const actorUrl = `${fullPublicHost}/api/activitypub?action=actor&name=${name}`
       decodeURIComponent(aliases[0]).should.equal(actorUrl)
       const firstLink = _.find(links, { rel: 'self' })
       decodeURIComponent(firstLink.href).should.equal(actorUrl)
     })
+
+    it('should accept an actor URL as resource', async () => {
+      const user = createUser({ fediversable: true })
```

Comment:
i dont think we need 3 duplicated tests (user shelf entity) for a format validation already defined in the unit test below. Since its not an core implementation. I would just leave one of those test in a specific `describe(''actor URL as resource'', () => {}`

Reply:
done in 7a6b25